mempool/mempool.go

```diff
@@ -985,7 +994,6 @@ func (mp *TxPool) TxDescs() []*TxDesc {
 // concurrent access as required by the interface contract.
 func (mp *TxPool) MiningDescs() []*mining.TxDesc {
 	mp.mtx.RLock()
-	defer mp.mtx.RUnlock()
 
```

Comment:
Same thing here regarding the lock around the critical section and spacing consistency.